data/.gitignore

```diff
@@ -5,3 +5,4 @@ force-https.txt
 last_update.txt
 no-cache.txt
 update.php
+tos.html
```

Comment:
Maybe consider providing a tos.example.html or something

Reply:
I think giving an example file is quite complicated since ToS can be very different across different instances while being too important to be based on some "default" document. I wanted to add a bit of documentation on this feature but I'm waiting for #2164 and #2491. I'm keeping a list of what I want to document after that. Would you still prefer that I add a note somewhere?

Reply:
What about a *lorem ipsum* just to provide a template?